python/paddle/v2/fluid/io.py

```diff
@@ -46,6 +46,9 @@ def is_parameter(var):
 
 
 def is_persistable(var):
+    if var.desc.type() == core.VarDesc.VarType.FEED_MINIBATCH or \
+       var.desc.type() == core.VarDesc.VarType.FETCH_LIST:
+        return False
```

Comment:
@Xreki  I have changed the code and go with option 3 using your suggestion. For option 2, there is problem. Because in the python side of the code, the operator `op` field of var will only be associated with the operator that have this variable as its output. So for feed variable, since it is not the output of any operator. Its `op` data member will be `None`.